BlackJack/index.js

```diff
@@ -29,189 +41,295 @@ makeCard();
 
 //onpage card
 let dealerRandomCard = Math.floor(Math.random() * allCard.length);
-dealerImg.src = `cards/${allCard[dealerRandomCard]}`
-dealerCards.appendChild(dealerImg)
-let dealerCardN = allCard[dealerRandomCard]
-let dealerCardNum = dealerCardN.charAt(0)
+dealerImg.src = `cards/${allCard[dealerRandomCard]}`;
+dealerCards.appendChild(dealerImg);
+let dealerCardN = allCard[dealerRandomCard];
+let dealerCardNum = dealerCardN.charAt(0);
 
 //dealer hidden card
 // let dealerHiddenRandomCard = Math.floor(Math.random() * allCard.length) +1;
 
-
 let playerRandomCard = Math.floor(Math.random() * allCard.length);
-playerImg.src = `cards/${allCard[playerRandomCard]}`
-playerCards.appendChild(playerImg)
-
-let playerRandomCard2 = Math.floor(Math.random() * allCard.length) +1;
-playerImg2.src = `cards/${allCard[playerRandomCard2]}`
-playerCards.appendChild(playerImg2)
+playerImg.src = `cards/${allCard[playerRandomCard]}`;
+playerCards.appendChild(playerImg);
```

Comment:
If you start to feel that you are copy pasting code or retyping the same logic. It means it is time for a reusable function.

This sample function will draw a card using the logic you have, and also the added logic of removing the card using splice, so you will use the index to draw and remove the same card.
`
function drawCard() {
  let randomIndex = Math.floor(Math.random() * allCard.length);
  let card = allCard[randomIndex];
  allCard.splice(randomIndex, 1); // Remove the drawn card from the deck
  return card;
}`